### DISCUSSION TITLE
prototype-p8 with spack-stack version 1.5.1?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#2426] [Q&A]
Can the prototype-p8 tag be compiled with spack-stack 1.5.1? If not, what is range of options? Thanks!

**Comment by gspetro-NOAA** [comment]
Hi Ben! 

It sounds from @jkbk2004 and @ulmononian like the prototype-p8 tag is pretty old (from 2022) before we supported spack-stack. In theory, you could modify a spack-stack template to match all the library versions used in the HPC-Stack modules from the p8 tag and then build spack-stack from there. Not sure if you want to be building a custom spack-stack though... :-/ 

Best,
Gillian

**Comment by jkbk2004** [comment]
@benjamin-cash I think CPC continues to run p8 tag on gaea: https://github.com/ufs-community/ufs-weather-model/issues/1755.
